### PR TITLE
BookStore App: Setup for project

### PR DESCRIPTION
**Pull request by AbayomiOlaoye**
### Project requirements

- [x] Initialise React app.
- [x] The building blocks of your app should be set as re-usable components. Create a directory for your components.
- [x] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
##### Books - the default view:
- [x] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
- [x] Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- [x] Should display a form for adding a book (no functionality yet).
- [x] Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
##### Categories:
- [x] Should display a button "Check status" only.
- [x] Styling is not required at this point.

### CODE REVIEW

**Review by victorokoroji — CHANGES_REQUESTED** [review]
Hi @AbayomiOlaoye, 

You have done a great job so far! 💪 👏
You still need to work on a few issues before moving on to the next project, but you're almost there!

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Required Changes ♻️

- [ ] I must commend your work because it is very impressive, but the project requirement specifies that your project should be initialized using create-react-app and not vite. Kindly implement it 👍🏽 

_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/ViewBook.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <p className="title">
+      Title:
+      {book.title}
+    </p>
+    <p className="author">
+      Author:
+      {book.author}
+    </p>
+  </div>
+);
```
Comment (mmsesay) [inline src/components/ViewBook.js:15]: - [OPTIONAL] Please add a button for removing books. 
       - The button wouldn't remove the books yet, this is just a placeholder for now.

**Review by mmsesay — APPROVED** [review]
Hi @AbayomiOlaoye, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
 - Good Readme :heavy_check_mark: 
 - No linter error :heavy_check_mark: 
 - Good PR title :heavy_check_mark: 
 - Good PR summary :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/ViewBook.js (hunk 1 above)
Comment (AbayomiOlaoye) [reply src/components/ViewBook.js:15]: Thank so much, @mmsesay

**Pull request merged**